apps/aecore/lib/aecore/structures/data_tx.ex

```diff
@@ -72,7 +72,13 @@ defmodule Aecore.Structures.DataTx do
           ChainState.chainstate()
   def process_chainstate!(%DataTx{} = tx, chainstate, block_height) do
     accounts_state = chainstate.accounts
-    tx_type_state = Map.get(chainstate, tx.type.get_chain_state_name(), %{})
+
+    tx_type_state =
+      if(tx.type == SpendTx) do
```

Comment:
Why not just return `nil` in `SpendTx.get_chain_state_name()`? It would work as expected.